include/vcpkg/base/util.h

```diff
@@ -142,6 +142,13 @@ namespace vcpkg::Util
         return std::find(begin(cont), end(cont), v);
     }
 
+    template<class Container, class V>
+    auto contains(Container&& cont, V&& v)
```

Comment:
```suggestion
    auto contains(const Container& cont, const V& v)
```

okay I know that the existing code does this but no need to do it in new code.